UWP/glslang_UWP/glslang_UWP.vcxproj

```diff
@@ -479,7 +479,7 @@
     <ClCompile Include="..\..\ext\glslang\SPIRV\Logger.cpp" />
     <ClCompile Include="..\..\ext\glslang\SPIRV\SpvBuilder.cpp" />
     <ClCompile Include="..\..\ext\glslang\SPIRV\SPVRemapper.cpp" />
-    <ClCompile Include="..\..\ext\glslang\SPIRV\SpvPostProcess.cpp" />
+    <!--<ClCompile Include="..\..\ext\glslang\SPIRV\SpvPostProcess.cpp" />-->

```

Comment:
I had to comment that, because at least when I updated the code, the build was complaining about not being able to find that file

Reply:
Got it, I'll keep the rebase in mind. Thanks, and sorry about that. I would've done a script to copy files, but didn't for two reasons:

1. VS/UWP needs this to be added as content.
2. I planned to allocate more time into this project next week to work on some other features I'm planning, by then I will do other PR to fix/improve some of this things and make PR for the new features separately.

Reply:
WTF? I didn't mean to do this... Is there anyway to clean this mess I just made? You know what? Reject this PR, I'll make a new one if possible.

Reply:
It looks like you rebased from an older commit and maybe merged.  Don't worry, this is easy to fix.

```sh
git checkout local
git remote add upstream https://github.com/hrydgard/ppsspp
git fetch upstream
git rebase -i upstream/master
# Delete the lines with any commits that aren't yours.
git push origin -f local
```

It's important to use the `-f` flag on push and not to do any `git merge` or `git pull` while rebasing.

-[Unknown]

Reply:
Awesome, that fixed it. Thanks for the help.

Reply:
Cool, but the commits that added and then removed the large binary files are still there, so the large files (if this was merged) would always bloat the size of the .git directory.

You can use `squash` in the rebase instruction list instead of `pick` to combine commits, or else `edit` to change them.

-[Unknown]

Reply:
Ok, so I found some time now to implement a prebuild command to copy files over. What I've been trying to do for a few hours is to fix the rebase. Whenever I run the rebase -i command, pick the commit I want and squash the others and save, it complains that it doesn't find the file ".git/rebase-merge/git-rebase-todo". If anyone has encountered that error and knows how to fix it, please let me know.

Reply:
Maybe [this](https://stackoverflow.com/questions/17140570) could help?

Reply:
Thanks @vnctdj that helped. I could rebase the commits, though, I destroyed my working copy. I'll fix it as soon as I get back home, commit latest changes. And hopefully fully comply with the requested changes.

Reply:
In case you don't know, deleted commits are not really "destroyed", they are kept back in the `reflog` for quite a long time :)

Reply:
Perfect, all seemed to have committed now. please, let me know if this fulfill the request. Thanks for the help